source/cluster-wide.rst

```diff
@@ -61,22 +67,24 @@ Kubernetes.
 
       $ kubectl create namespace pxc
 
-#. Edit the ``deploy/cw-bundle.yaml`` configuration file to set proper namespaces:
+   .. only:: comment
+
+      #. Edit the ``deploy/cw-bundle.yaml`` configuration file to set proper namespaces:
 
-   .. code:: yaml
+         .. code:: yaml
 
-      ...
-      subjects:
-      - kind: ServiceAccount
-        name: percona-xtradb-cluster-operator
-        namespace: "pxc-operator"
-      ...
-      env:
-               - name: WATCH_NAMESPACE
-                 value: "pxc"
-      ...
+            ...
+            subjects:
+            - kind: ServiceAccount
+              name: percona-xtradb-cluster-operator
+              namespace: "pxc-operator"
```

Comment:
Actually editing this part in cw-bundle.yaml will need to stay because otherwise if you change the operator namespace to something other then `pxc-operator` the pod will be created in this other namespace, but we will get permission errors.

Reply:
But now I see `Operator's namespace should be the ``pxc-operator`` one.`
@hors Is the operators namespace fixed to pxc-operator?

Reply:
@tplavcic if you want to change it you need to change namespace: "pxc-operator" in source/cluster-wide.rst file  too

Reply:
@hors Yes, but is that something that we want to mention to the user or not? As it is currently stated it should not be changed, so I'll approve and leave it at that. We can change later if there will be need.